src/component/elements/Input.tsx

```diff
@@ -107,20 +122,20 @@ const Input = forwardRef(
       nullable = false,
       onClear,
       datalist = [],
-      ...props
-    }: InputProps,
-    ref: ForwardedRef<HTMLInputElement>,
-  ) => {
+      ...otherProps
+    } = props;
     const [val, setVal] = useState(value);
     const valueRef = useRef(value);
     const localRef = useRef<any>();
     const combinedRef = useCombinedRefs([ref, localRef]);
 
-    const debounceOnChange = useRef(
-      debounce((value) => {
-        onChange(value);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This is the most important rule in a React project. If you have to disable it, it means you are leaving a bug in your code.

Reply:
I think we could useMemo instead of useCallback which returns a function 
- #2297

Reply:
Thanks, that PR looks good to me.